benchmarks/utils/update_artifacts.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import os
+import shutil
+
+BENCHMARK_REPORT_PATH = "/tmp/ts_benchmark"
+BENCHMARK_ARTIFACTS_PATH = "/tmp/ts_artifacts"
+BENCHMARK_REPORT_FILE = "ab_report.csv"
+WINDOW_LEN = 8
+WINDOW_START = 0
+
+# Copy BENCHMARK_REPORT_FILE to artifacts
+def copy_benchmark_reports(input, output):
```

Comment:
isn't this a tree copy? I figured standard libs might already implement this for us